dev-aws/kafka-shared-msk/account-identity/__env.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
```

Comment:
please symlink this to https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/dev-aws/kafka-shared-msk/__env.tf